File: app/src/main/java/com/example/liars_dice/model/game/Bet.java
package com.example.liars_dice.model.game;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Bet {
    private int value;
    private int amount;

    public Bet(
            @JsonProperty("value") int value,
            @JsonProperty("amount") int amount) {
        this.value = value;
        this.amount = amount;
    }

    public int getValue() {
        return value;
    }

    public void setValue(int value) {
        this.value = value;
    }

    public int getAmount() {
        return amount;
    }

    public void setAmount(int amount) {
        this.amount = amount;
    }
}

File: server/src/models/game/bet.ts
export class Bet {
    private value: number; // value on dice
    private amount: number; // number of dies with that value

    constructor(value: number, amount: number) {
        this.value = value;
        this.amount = amount
    }

    public getValue() {
        return this.value;
    }

    public getAmount() {
        return this.amount;
    }
}